Modified the initial beam size in the IOTA lens benchmark example.

cemitch99 opened the pull request:
The previous beam size was too large, resulting in a few particles experiencing large changes (>10%) in the quantities that should be invariants of motion.  The maximum change is now ~10^(-3).

cemitch99 committed:
Modified the initial beam size in the IOTA lens benchmark example.

ax3l commented:
Restarting CI (had some hick up)

ax3l commented:
Awesome, thanks!
@cemitch99 do you also want to edit the Python test script to make the tolerance more narrow? :)

cemitch99 committed:
Modified tolerance for IOTA lens benchmark example.

Reduced tolerance to account for reduced beam size and improved
preservation of the invariants of motion.

ax3l reviewed (approved):
Awesome, thank you! :sparkles:

ax3l commented on examples/iota_lens/analysis_iotalens.py:
```diff
@@ -96,20 +96,20 @@ def read_all_files(file_pattern):
 # add new columns: dH and dI
-beam_joined['dH'] = beam_joined["H_initial"] - beam_joined["H_final"]
-beam_joined['dI'] = beam_joined["I_initial"] - beam_joined["I_final"]
+beam_joined['dH'] = abs(beam_joined["H_initial"] - beam_joined["H_final"])
+beam_joined['dI'] = abs(beam_joined["I_initial"] - beam_joined["I_final"])
```
If we use the `.abs()` method of the resulting dataframe, then pandas might be able to compute this in delayed fashion (when we access it later - aka lazy computing).
```suggestion
beam_joined['dH'] = (beam_joined["H_initial"] - beam_joined["H_final"]).abs()
beam_joined['dI'] = (beam_joined["I_initial"] - beam_joined["I_final"]).abs()
```

ax3l committed:
Small update: `.abs()`
